doc/API/ResourceServiceDetail.yaml

```diff
@@ -0,0 +1,364 @@
+Monitoring.ServiceDetail:
+  type: object
+  properties:
+    uuid:
+      type: string
+      description: "Unique identifier of the Service resource"
+      example: "h14"
+    id:
+      type: integer
+      format: int32
+      description: "Id of the Service resource"
+      example: 14
+    name:
+      type: string
+      description: "Name of the Service resource"
+      example: "Ping"
+    monitoring_server_name:
+      type: string
+      description: "Name of the monitoring server that monitors the Service resource"
+      example: "Central"
+    type:
+      type: string
```

Comment:
enums could be used : https://swagger.io/docs/specification/data-models/enums/

Reply:
Fixed ✅

Reply:
not fixed :)
But we've decided to make a global update about enums in a separate PR

Reply:
Yes I did not changed it here but for the possible statuses